test/snapshots/markdownlint-test-repos.js.md

```diff
@@ -8,7 +8,7 @@ Generated by [AVA](https://avajs.dev).
 
 > Expected linting violations
 
-    ''
+    'test-repos/apache-airflow/tests/system/providers/apache/hive/example_twitter_README.md: 36: MD045/no-alt-text Images should have alternate text (alt text) [Element: img]'
```

Comment:
Fewer than I thought! Please let me know if you did not visually verify each one of these and I will.